x-pack/plugin/sql/qa/src/main/resources/geo/docs.csv-spec

```diff
@@ -10,9 +10,18 @@
 
 selectAsWKT
 // tag::aswkt
-SELECT city, ST_ASWKT(location) location FROM "geo" WHERE city = 'Amsterdam';
+SELECT city, ST_AsWKT(location) location FROM "geo" WHERE city = 'Amsterdam';
 
      city:s    |     location:s
 Amsterdam      |point (4.850311987102032 52.347556999884546)
 // end::aswkt
 ;
+
+selectAsWKT
+// tag::wkttosql
+SELECT ST_AsWKT(ST_WKTToSQL('POINT (10 20)')) location;
```

Comment:
Why do you have to wrap it with `ST_AsWKT`?

Reply:
In this test we compare the result with csvjdbc result. The csvjdbc driver cannot deal with geoshapes. So, I have to convert the column into some type that csvjdbc understands.